tests/wpt/mozilla/tests/css/issue_18544_flow_list_crash.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Issue 18544 - Flow list crash</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div style="position:fixed"></div>
+<script>
+  async_test(function(test) {
+    window.onload = test.step_func_done(function() {
+      document.body.children[0].style['column-count'] = 1;
```

Comment:
let's do `document.body.offsetTop;` below this line so we're sure that layout has been triggered before reporting the test as ok.